bindgen/ir/comp.rs

```diff
@@ -586,6 +595,7 @@ where
     let mut unit_size_in_bits = 0;
     let mut unit_align = 0;
     let mut bitfields_in_unit = vec![];
+    let mut bitfields_bit_begin_off = 0;
```

Comment:
Could you add a comment for what this tracks? Something like `// Offset into the previous member, where the bitfields begin at.`